packages/next/lib/verifyEslintSetup.ts

```diff
@@ -0,0 +1,99 @@
+import chalk from 'chalk'
+import { ESLint } from 'eslint'
+import { promises } from 'fs'
+import { join } from 'path'
+
+import { formatResults } from './eslint/customFormatter'
+import { fileExists } from './file-exists'
+import * as log from '../build/output/log'
+
+import findUp from 'next/dist/compiled/find-up'
+
+type Config = {
+  plugins: string[]
+}
+
+export async function verifyEslintSetup(
+  baseDir: string,
+  pagesDir: string,
+  pagePath: string | null
+) {
+  try {
+    let options: ESLint.Options
+
+    let pathNotExists = Boolean(
+      pagePath && !(await fileExists(join(pagesDir, pagePath)))
+    )
+
+    if (pathNotExists) return
+
+    const dirResults = await promises.readdir(baseDir)
+    const eslintrcFile = dirResults.find((file: string) =>
```

Comment:
Should we findUp the eslintrc too? Mono repo setups could potentially have lint config at a higher level

Reply:
Good call, updated :)